src/index.html

```diff
@@ -5,8 +5,423 @@
+                class="nav__link"
+                href="#samsung"
+              >
+                Samsung
+              </a>
+            </li>
+            <li class="nav__items">
+              <a
+                class="nav__link"
+                href="#smartphones"
+              >
+                Smartphones
+              </a>
+            </li>
+            <li class="nav__items">
+              <a
+                class="nav__link"
+                data-qa="nav-hover"
+                href="#laptops&computers"
+              >
+                Laptops & computers
+              </a>
+            </li>
+            <li class="nav__items">
+              <a
+                class="nav__link"
+                href="#gadgets"
+              >
+                Gadgets
+              </a>
+            </li>
+            <li class="nav__items">
+              <a
+                class="nav__link"
+                href="#tablets"
+              >
+                Tablets
+              </a>
+            </li>
+            <li class="nav__items">
+              <a
+                class="nav__link"
+                href="#photo"
+              >
+                Photo
+              </a>
+            </li>
+            <li class="nav__items">
+              <a
+                class="nav__link"
+                href="#video"
+              >
+                Video
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+
+      <main class="main">
+        <div class="product product--catalog">
+          <div class="product__card" data-qa="card">
+            <div class="product__box">
+              <img
+                class="product__image"
+                src="./images/imac.jpeg"
+                alt="product image"
+              >
+
+              <h2 class="product__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+
+              <p class="product__fonts product__fonts--code">Product code: 195434</p>
+
+              <div class="product__stars">
+                <div class="stars-block">
+                  <ul class="stars stars--4">
+                    <li class="stars__star"></li>
+                    <li class="stars__star"></li>
+                    <li class="stars__star"></li>
+                    <li class="stars__star"></li>
+                    <li class="stars__star"></li>
+                  </ul>
+                </div>
+                <p class="product__fonts product__fonts--reviews">Reviews: 5</p>
+              </div>
+
+              <div class="product__price">
+                <p class="product__fonts product__fonts--price">Price:</p>
+                <p class="product__fonts product__fonts--calculation-price">$2,199</p>
+              </div>
+
+              <a
+                class="product__button"
+                href="#"
+                data-qa="card-hover"
+              >
+                Buy
+              </a>
+
+            </div>
+          </div>
```

Comment:
no needs to add an empty line between parent tag and child one
```suggestion
              </a>
            </div>
          </div>
```